### PR TITLE
Added reactotron connection fix for ios devices

### PR DESCRIPTION
**Pull request by guidoprinc**
Searched in reactotron discussions and added this lines to fix ios ( with physical devices) connection with reactotron.

### CODE REVIEW

File: generators/app/templates/src/config/ReactotronConfig.ejs [hunk 1]
```diff
@@ -3,10 +3,15 @@ import Reactotron, { trackGlobalErrors } from 'reactotron-react-native';
 import apisaucePlugin from 'reactotron-apisauce';
 import { reactotronRedux } from 'reactotron-redux';
 
+// If you want to use a physical device and connect it to reactotron, execute first `adb reverse tcp:9090 tcp:9090`
+let scriptHostname;
 if (__DEV__) {
+  const scriptURL = NativeModules.SourceCode.scriptURL;
+  scriptHostname = scriptURL.split('://')[1].split(':')[0];
```
Comment (sbalay) [inline generators/app/templates/src/config/ReactotronConfig.ejs]: do we always want this or only for ios?
Why is this needed?

Comment (guidoprinc) [reply generators/app/templates/src/config/ReactotronConfig.ejs]: when i wanted to check redux status with QA's ios device, i couldn't get a connection with reactotron. I checked reactotron issues and got that ios uses the owner ip to connect with the device in this issue: https://github.com/infinitered/reactotron/issues/272
For android it works when using the comment above.

File: generators/app/templates/src/config/ReactotronConfig.ejs [hunk 2]
```diff
@@ -3,10 +3,15 @@ import Reactotron, { trackGlobalErrors } from 'reactotron-react-native';
...
   Reactotron.configure({ name: '<%= projectName %>' })
     .use(trackGlobalErrors())
     .use(apisaucePlugin())
     .use(reactotronRedux({ onRestore: state => Immutable(state) }))
     .connect();
 }
+Reactotron.configure({ host: scriptHostname }).connect();
```
Comment (sbalay) [inline generators/app/templates/src/config/ReactotronConfig.ejs]: this line is duplicated with line 11. We're already executing configure and connect.

Comment (guidoprinc) [reply generators/app/templates/src/config/ReactotronConfig.ejs]: will add it to previous line

File: generators/app/templates/src/config/ReactotronConfig.ejs [hunk 3]
```diff
@@ -2,9 +2,14 @@ import Immutable from 'seamless-immutable';
...
 import apisaucePlugin from 'reactotron-apisauce';
 import { reactotronRedux } from 'reactotron-redux';
+import { NativeModules } from 'react-native';
 
+// If you want to use a physical device and connect it to reactotron, execute first `adb reverse tcp:9090 tcp:9090`
+let scriptHostname;
 if (__DEV__) {
-  Reactotron.configure({ name: '<%= projectName %>' })
+  const scriptURL = NativeModules.SourceCode.scriptURL;
+  scriptHostname = scriptURL.split('://')[1].split(':')[0];
```
Comment (sbalay) [inline generators/app/templates/src/config/ReactotronConfig.ejs]: remove the let outside the if and declare this variable with const
